src/reference/asciidoc/jms.adoc

```diff
@@ -397,6 +397,23 @@ __NOT ALLOWED__
 User-supplied correlation ids are not permitted with a reply listener; the gateway will not initialize with this
 configuration.
 
+[[jms-async-gateway]]
+==== Async Gateway
+
+Starting with _version 4.3_, you can now specify `async="true"` (or `setAsync(true)`) when configuring the outbound
+gateway.
+
+By default, when a request is sent to the gateway, the requesting thread is suspended until the reply is received and
+the flow then continues on that thread.
+If `async` is true, the requesting thread is released immediately after the send completes, and the reply is returned
+(and the flow continues) on the listener container thread.
+This can be useful when the gateway is invoked on a poller thread; the thread is released and is available for other
+tasks within the framework.
+
+`async` requires a `<reply-listener/>` (or `setUseReplyContainer(true)` when using Java configuration); it also
```

Comment:
Maybe "The `async`"?
Otherwise it looks a bit weird to star the sentence from the lower case.